### ISSUE TITLE
32-bit Linux mega-cmd-server hangs on startup

### ISSUE THREAD

**Issue by patrick-brian-mooney** [meganz/MEGAcmd#212]
Just installed mega-cmd-server on an old laptop that's recently been wiped and had a fresh install of Ubuntu Server 16.04. When running it for the first time, it prints its banner and then "Listening to petitions," then hangs completely. It never tries to authenticate, nor does it prompt for a command, nor does it ever print anything else out, no matter how long I wait. It quits immediately on a ctrl-C. 

Trying to follow advice in issue #122, I tried `rm ~/.megaCmd/megaclient_statecache*`, but there's not any cache files there yet. I've attached the log created with `mega-cmd-server --debug-full | tee ~/mega.log`.

**Comment by polmr** [comment]
Attached? Did you send it to support@mega.nz referencing this issue?

**Comment by patrick-brian-mooney** [comment]
Whoops. No. I was perfectly convinced I'd actually attached it. Let me try again.


[mega.log](https://github.com/meganz/MEGAcmd/files/3298593/mega.log)

**Comment by polmr** [comment]
no problem @patrick-brian-mooney , I've been trying to reproduce it in a Ubuntu 16.04. It works flawlessly, and I'm afraid the log does not contains much info, rather that it seems to start ok

Just to be sure there's an issue here. Are you aware that you don't need to execute the server directly? You'd rather execute the interactive shell (in linux `mega-cmd`) once you have MEGAcmd installed?
If you have any doubt, please read https://github.com/meganz/MEGAcmd, specially https://github.com/meganz/MEGAcmd#linux. And also you might want to take a look at the user guide here: https://github.com/meganz/MEGAcmd/blob/master/UserGuide.md

If there's something unclear there do not hesitate to tell us.

If there's an actual issue and the server is crashing, we might want to debug it a little bit, if you wish to help us:

1 - install debug symbols 
sudo apt update && sudo apt install megacmd-dbg

2 - run mega-cmd-server in a debugger (you might need to install gdb)
gdb mega-cmd-server
It should say something like:
```Reading symbols from mega-cmd-server...Reading symbols from /usr/lib/debug/.build-id/10/3dc7527ee44c6531970f4ab228264c223bc4c6.debug...done.
done.
(gdb) 
```
3 - Then run the program (type 'r' + Enter)
Let it run until it crashes. (if it doesn't, open another console and use the interactive shell `mega-cmd` until it crashes)

4 - once so, then type in the gdb console: 'thread apply all bt'
Press "Enter" until the output is complete, and 

5 - then please, scroll back and send us the entire output of that command.
